### PR TITLE
[ADD] Travis config

### PR DESCRIPTION
**Pull request by StefanRijnhart**
Missed this when porting the framework

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - flake8 odoo/addons/openupgrade* --max-line-length=120 --exclude=__init__.py
+# only flake8 migration scripts from the openupgrade project, presumably
+# identifyable by using the openupgrade helpers
+    - flake8 --max-line-length=120 $(find . \( -name 'pre-*.py' -or -name 'post-*.py' \) -exec grep -q openupgrade {} \; -print)
+    - flake8 odoo/addons/*/migrations/*/tests/ --max-line-length=120
+    - flake8 addons/*/migrations/*/tests/ --max-line-length=120
+    - npm install -g less less-plugin-clean-css
+    - createdb $DB
+    - wget -q -O- https://github.com/OCA/OpenUpgrade/releases/download/databases/10.0.psql | pg_restore -d $DB --no-owner
+    - cat odoo/addons/base/migrations/*/tests/*.yml addons/*/migrations/*/tests/*.yml > ../test_data90.yml || true
```
Comment (yvaucher) [inline .travis.yml]: Just nitpicking

test_data100.yml ? or maybe lets give it a generic name instead of renaming it on each version?

**Review by yvaucher — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
v11 uses python3

**Comment by StefanRijnhart** [comment]
@yvaucher @mreficent thanks, fixed.

**Comment by MiquelRForgeFlow** [comment]
and pip3

**Comment by StefanRijnhart** [comment]
@mreficent thanks again. I have never worked with Python 3 so please excuse my ignorance.

**Comment by StefanRijnhart** [comment]
@mreficent I am reverting to running travis with Python 2.7, because the OpenUpgrade code is not yet compatible with Python 3.5 (while Odoo itself is still compatible with Python 2.7 although it is not officially supported). I will start working on porting the code immediately and propose in a separate PR.

**Comment by yvaucher** [comment]
The 2 other errors are because of the missing database as sql file.

```
The command "wget -q -O- https://github.com/OCA/OpenUpgrade/releases/download/databases/10.0.psql | pg_restore -d $DB --no-owner" exited with 1.
```
```
The command "psql $DB -c "update ir_module_module set state='uninstalled' where name not in ('$(echo $MODULES | sed -e "s/,/','/g")')"" exited with 1.
```

**Comment by yvaucher** [comment]
Merging as a first step (need this for py3 compatibility)

**Pull request merged**